docs/source/developers/cpp/windows.rst

```diff
@@ -362,6 +362,21 @@ suppress dllimport/dllexport marking of symbols. Projects that statically link
 against Arrow on Windows additionally need this definition. The Unix builds do
 not use the macro.
 
+Downloading the Timezone Database
+=================================
+
+To run some of the compute unit tests on Windows, the IANA timezone database
+and the Windows timezone mapping need to be downloaded first. To download
+them to the default search directory (in the user Downloads directory), use 
```

Comment:
Can you just cross-reference to the relevant section in the building docs above?

Reply:
Yeah that makes sense 👍